app/models/user.rb

```diff
@@ -198,6 +200,10 @@ def set_current_language
     self.language = I18n.locale.to_s if self.language.blank?
   end
 
+  def set_default_color_theme
+    self.color_theme ||= DEFAULT_COLOR_THEME
+  end
+
```

Comment:
Oh well, I guess since we do that for the locale too we can leave figuring that out to another time.